osu.Game/Screens/Menu/IntroCircles.cs

```diff
@@ -19,7 +19,7 @@ public class IntroCircles : IntroScreen
 
         protected override string BeatmapFile => "circles.osz";
 
-        public const double TRACK_START_DELAY = 600;
+        public const double TRACK_START_DELAY = 1000;
```

Comment:
Without even running the code, I can tell you this is going to be wrong because it's not accounting for the actual circles theme being played (the more common case).

Reply:
Ah that's correct. So after I revert that, Should I increase the delay even further?